docs/software/integrations/index.mdx

```diff
@@ -9,6 +9,8 @@ The Meshtastic ecosystem is highly extensible and allows easy integration with a
 
 Current Meshtastic integrations:
 
+- [MQTT](/docs/software/integrations/MQTT/) - Expands Meshtastic's capabilities by connecting with MQTT brokers, facilitating data exchange and integration with platforms such as Mosquitto, Python scripts, Node-RED, Home Assistant, and Adafruit IO.
```

Comment:
`/docs/software/integrations/MQTT/`  While `MQTT` being capital here technically works, I think we're better off keeping everything lowercase.